firestore.rules

```diff
@@ -1,7 +1,11 @@
+rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
-    match /{document=**} {
-      allow read, write: if false;
+    match /apiKeys/{document=**} {
+      allow read, write: if request.auth.token.email.split("@")[1] == "actnowcoalition.org";
```

Comment:
So I'm not 100% sure but because we have email/password sign-in enabled, I think a malicious user could create a foo@actnowcoalition.org account (e.g. by making direct API calls to firebase auth) and then be signed in with an @actnowcoalition.org account and this rule would give them full access.

To prevent this we could:
1. Disable email/password auth if we don't need it, or
2. Change this rule to additionally check for `request.auth.token.firebase.sign_in_provider == "google.com"` so it only allows people using google sign-in, or
3. Change this rule to additionally check for `request.auth.token.email_verified` which will only be true if they've verified their email address.  I'm guessing this will be true automatically if they're signed in with google auth, but I haven't verified.

Reply:
ah, good call. I've added `request.auth.token.email_verified` and will remove the standard email login from both projects